packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -242,17 +243,15 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
   return (
     <Stack verticalFill grow styles={compositeOuterContainerStyles} id={compositeParentDivId}>
       <Stack horizontal grow>
-        {!isMobileWithActivePane && (
```

Comment:
Can you test also, in mobile, does the PiPiP still work as expected? I'm worried the video tiles on the gallery will still have handles to the streams and maybe won't load the streams of the PiPiP correctly

Reply:
The Call local stream stops working after opening and closing the chat/people pane.
https://user-images.githubusercontent.com/79475487/164388812-4d1c3dbc-66af-4ab9-98ca-5e988ffde635.mp4
The fix will to be done to the streams in the Call without rerendering the Call. 

Closing this PR. 

Logged a bug for the Compliance Banner reappearing https://skype.visualstudio.com/SPOOL/_search?text=Dismissed*&type=workitem&lp=workitems-Project&filters=Projects%7BSPOOL%7D&pageSize=25.